api/src/test/java/org/openmrs/module/fhir2/providers/r3/TaskFhirResourceProviderTest.java

```diff
@@ -170,25 +188,24 @@ public void getTaskHistoryByWithWrongId_shouldThrowResourceNotFoundException() {
 	//		resourceProvider.updateTask(new IdType().setValue(WRONG_TASK_UUID), wrongTask);
 	//	}
 	
-	//TODO ERROR
-	//	@Test
-	//	public void searchTasks_shouldReturnMatchingTasks() {
-	//		List<org.hl7.fhir.r4.model.Task> tasks = new ArrayList<>();
-	//		tasks.add(task);
-	//
-	//		when(taskService.searchForTasks(any(), any(), any(), any())).thenReturn(tasks);
-	//
-	//		TokenOrListParam status = new TokenOrListParam();
-	//		TokenParam statusToken = new TokenParam();
-	//		statusToken.setValue("ACCEPTED");
-	//		status.add(statusToken);
-	//
-	//		Bundle results = resourceProvider.searchTasks(null, null, status, null);
-	//
-	//		assertThat(results, notNullValue());
-	//		assertThat(results.getTotal(), equalTo(1));
-	//		assertThat(results.getEntry(), notNullValue());
-	//		assertThat(results.getEntry().get(0).getResource().fhirType(), equalTo("Task"));
-	//		assertThat(results.getEntry().get(0).getResource().getId(), equalTo(TASK_UUID));
-	//	}
```

Comment:
I guess you forgot about this. You should probably remove

Reply:
Oh yeah, I think I forgot.

Reply:
I have added the correct test for this just below it. Hence deleted.